.github/CODEOWNERS

```diff
@@ -3,39 +3,26 @@
 # For details on acceptable file patterns, please refer to the [Github Documentation](https://help.github.com/articles/about-codeowners/)
 
 # default owners, overridden by package specific owners below
-* @asaj @cmcewen @timmoreton
+* @celo-org/default-owners
 
 # directory and file-level owners. Feel free to add to this!
-/packages/azure/ @codyborn
-/packages/blockchain-api/ @jmrossy @cmcewen @i1skn @jeanregisser
-/packages/celotool/ @asaj
-/packages/cli/ @mcortesi @gastonponti @medhak1 @celo-org/devx
-/packages/sdk/ @mcortesi @gastonponti @medhak1 @barbaraliau @celo-org/devx
-/packages/dappkit/ @cmcewen @barbaraliau @celo-org/devx
-/packages/dev-utils/ @mcortesi
-/packages/docs/ @timmoreton @barbaraliau @critesjosh 
-/packages/faucet/ @eelanagaraj @celo-org/devx
-/packages/mobile/ @celo-org/wallet 
-/packages/moonpay-auth/ @annakaz @i1skn 
-/packages/notification-service/ @annakaz @cmcewen @i1skn @jeanregisser
-/packages/flake-tracker @alecps
-
-/packages/protocol/ @celo-org/cap
-/packages/sdk/contractkit/ @celo-org/cap
-/packages/komencikit/ @celo-org/cap
-/packages/phone-number-privacy/ @celo-org/cap
-/packages/attestation-proxy/ @celo-org/cap
-/packages/attestation-service/ @celo-org/cap
-/packages/metadata-crawler/ @celo-org/cap
+/packages/attestation-proxy/ @celo-org/identity
+/packages/attestation-service/ @celo-org/identity
+/packages/azure/ @celo-org/azure
+/packages/celotool/ @celo-org/protocol-owners
+/packages/cli/ @celo-org/applications
+/packages/dev-utils/ @celo-org/applications
+/packages/docs/ @celo-org/applications @celo-org/devrel
 /packages/env-tests @celo-org/cap
-
-/packages/react-components/ @i1skn @jeanregisser @celo-org/wallet
-/packages/terraform-modules-public/ @tkporter
-/packages/terraform-modules/ @tkporter
-/packages/typescript/ @cmcewen
-/packages/sdk/utils/ @barbaraliau @medhak1 @celo-org/devx
-/packages/sdk/base/ @barbaraliau @medhak1 @celo-org/devx
-
-# helm-charts rules.
-/packages/helm-charts/ @nambrot @timmoreton
-/packages/helm-charts/oracle/ @celo-org/platform-economics
+/packages/faucet/ @celo-org/applications
+/packages/flake-tracker @celo-org/flake-trackers
+/packages/helm-charts/ @celo-org/helm-chart-owners
```

Comment:
@aaronmgdr  I'm curious as to why there are separate teams for helm and terraform, instead of a single DevOps team owning this? Do you know?

Reply:
I did it that way because I there were different individuals owning these before and this was just a way of putting them into a group. 

if it makes more sense to have one group im all in. @lvpeschke thoughts?

Reply:
My take on this:
- @celo-org/helm-chart-owners as a group doesn't make sense
- helm charts should be owned by the team owning the rest of the application
- given the current structure, IMO it's okay for devops to be marked as code owners and to be responsible for making sure the right people are roped in for reviews